teachertool/src/style.overrides/Modal.scss

```diff
@@ -0,0 +1,13 @@
+
+.common-modal {
+    max-height: 90vh;
+    display: flex;

```

Comment:
Is it not already a flex? I'm surprised.

Reply:
I was shocked!